addons/mail/static/src/components/autocomplete_input/autocomplete_input.js

```diff
@@ -64,119 +68,21 @@ export class AutocompleteInput extends Component {
         if (this.root.el.contains(node)) {
             return true;
         }
-        if (!this.props.customClass) {
+        if (!this.autocompleteInputView.customClass) {
             return false;
         }
-        const element = document.querySelector(`.${this.props.customClass}`);
+        const element = document.querySelector(`.${this.autocompleteInputView.customClass}`);
         if (!element) {
             return false;
         }
         return element.contains(node);
     }
 
-    focus() {
-        if (!this.root.el) {
-            return;
-        }
-        this.root.el.focus();
-    }
-
-    //--------------------------------------------------------------------------
-    // Handlers
-    //--------------------------------------------------------------------------
-
-    /**
-     * @private
-     * @param {FocusEvent} ev
-     */
-    _onAutocompleteFocus(ev) {
-        if (this.props.focus) {
-            this.props.focus(ev);
```

Comment:
the focus part is gone?

Reply:
`_onAutocompleteFocus` is unused and `focus` becomes setting field `doFocus`.